include/swift/Option/Options.td

```diff
@@ -411,7 +411,13 @@ def enable_library_evolution : Flag<["-"], "enable-library-evolution">,
 
 def require_explicit_availability : Flag<["-"], "require-explicit-availability">,
   Flags<[FrontendOption, NoInteractiveOption]>,
-  HelpText<"Require explicit availability on public declarations">;
+  HelpText<"Warn on public declarations without an availability attribute">;
+
+def require_explicit_availability_EQ : Joined<["-"], "require-explicit-availability=">,
```

Comment:
I was considering renaming the flag to `-require-availability`, just to make it shorter even if a bit less descriptive.

Interesting idea for SPI, in most cases SPI doesn't require any availability so I don't think we'd need the "all SPI" version, but for specific SPI groups it could be useful and a good practice to apply. Users would have to learn to mark their features as SPI first before upgrading them to API instead of the other way around though.